### PR TITLE
Updating edge add-on naming to  Microsoft Compliance Extension

### PR DESCRIPTION
**Pull request by tdoucett-ms**
Microsoft Insider Risk Extension is now Microsoft Compliance Extension

@robmazz for review.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/insider-risk-management-browser-support.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The following table summarizes detected activities and extension support for eac
 
 ## Common requirements
 
-Before installing either the *Microsoft Insider Risk Extension* or the *Microsoft Compliance Extension,* customers need to ensure that devices for in-scope policy users meet the following requirements
+Before installing the Microsoft Edge add-on or Google Chrome extension, customers need to ensure that devices for in-scope policy users meet the following requirements
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-browser-support.md]:
```suggestion
Before installing the Microsoft Edge add-on or Google Chrome extension, customers need to ensure that devices for in-scope policy users meet the following requirements:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tdoucett-ms, kindly commit the minor additional suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@tdoucett-ms Gentle reminder. Please commit the suggested changes. Thanks

**Pull request merged**